target/linux/bcm27xx/patches-6.6/950-0487-drivers-media-imx296-Add-standby-delay-during-probe.patch

```diff
@@ -15,9 +15,9 @@ Signed-off-by: Naushir Patuck <naush@raspberrypi.com>
 
 --- a/drivers/media/i2c/imx296.c
 +++ b/drivers/media/i2c/imx296.c
-@@ -940,6 +940,8 @@ static int imx296_identify_model(struct
- 		return ret;
- 	}
+@@ -942,6 +942,8 @@ static int imx296_identify_model(struct
+ 
+ 	usleep_range(2000, 5000);
  
 +	usleep_range(2000, 5000);
```

Comment:
upstreamed patch.

Reply:
Odd that the update script did not flag this but thanks for pointing it out. Added.